SwanCustomEnvironments/swancustomenvironments/serverextension.py

```diff
@@ -31,37 +33,28 @@ async def get(self):
         self.set_header("Content-Type", "text/event-stream")
         makenv_process = SwanCustomEnvironmentsApiHandler.makenv_process
         
-        log_file = None
         if makenv_process is None:
-            # The first get request will launch the execution of the makenv.sh script,
-            # to create the environment
-            makenv_process = self._launch_makenv()
```

Comment:
You could maybe keep this call as it is, just to get the makenv process. And after that you launch the reading (_process_first_get).